### ISSUE TITLE
Announced on Delete is not working.

### ISSUE THREAD

**Issue by KishorePatel2005** [microsoft/fluentui#16542]
### Environment Information

- **Package version(s)**: 7.155.4
- **Browser and OS versions**: 87.0.4280.141   Windows 10 Enterprise 20H2

### Please provide a reproduction of the bug in a codepen:

Codepen Link :- https://developer.microsoft.com/en-us/fluentui#/controls/web/announced/quickactions

Vedio :- https://web.microsoftstream.com/video/2c5fa1ff-0400-9887-cead-f1eb5a47bb2a

#### Actual behavior:

It will not announceing deleteing on click of delete button on any item.

#### Expected behavior:

It should announced deleteing on click of delete button on any item.

### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: Normal

**Comment by gouttierre** [comment]
Thanks for the feedback and providing a recording of the issue at hand. 

@khmakoto  - Would you be able to determine if this is a regression or if this behavior is an issue when the delete button is clicked?  - I am not sure if this has to do with the Announce status being in-progress (deleting) and the time is short, therefore it does not reach the specified threshold.
cc @smhigley

**Comment by khmakoto** [comment]
@gouttierre this is not an issue, `Announced` does read that the item was deleted if you let Narrator announce everything regarding the surface it returns to after deletion first. This happens because Narrator first reads the new focused space and comments `Item deleted` after that.

@KishorePatel2005 could you retry again letting Narrator read everything and commenting back here if you are still encountering the issue?
